plugin/pkg/admission/resourcequota/admission_test.go

```diff
@@ -141,20 +119,15 @@ func TestAdmissionIgnoresDelete(t *testing.T) {
 	defer close(stopCh)
 
 	kubeClient := fake.NewSimpleClientset()
-	informerFactory := informers.NewSharedInformerFactory(kubeClient, controller.NoResyncPeriodFunc())
-	quotaAccessor, _ := newQuotaAccessor()
```

Comment:
what about adding the access to quotaAccessor just for the testing purpose? that way, the change to this test is kept to minimal.

Reply:
Let me explain the motivation for splitting this: 
1. In the current [admission_test.go](https://github.com/kubernetes/kubernetes/blob/61774cd7176cae0c0324d23ab20e6c6b3038153f/plugin/pkg/admission/resourcequota/admission_test.go) I saw that access to `quotaAccessor` is only needed for the test `TestAdmitPodInNamespaceWithoutQuota` because it modifies the livelookup cache.
2. IIUC, unit testing the lookup cache is a separate concern which is why I pulled out the lookup test specifics and put it to `staging/src/k8s.io/apiserver/pkg/admission/plugin/resourcequota/resource_access_test.go`.
3. Now the quotaAccessor remains an implementation detail of the admission plugin (was private before) and users potentially don't have to bother with it --> minimal surface.

WDYT?

> what about adding the access to quotaAccessor just for the testing purpose?

I can also make the accessor "public" but just doing this for a test didn't seem appropriate to me. Did you imagine another way?